src/proof/JsonWebSignatureProof.ts

```diff
@@ -134,12 +133,9 @@ export default class JsonWebSignatureProof implements Proof<JsonWebKey2020> {
             throw new Error(`No document found for  ${did}`)
         }
 
-        const foundKey = doc.verificationMethod?.find((pk: VerificationMethod) => pk.id.startsWith(iri));
-
-        if(!foundKey) {
-            throw new Error(`No Verification Method found for ${iri}`);
-        }
+        const foundKey = doc.verificationMethod?.find((pk: any) => pk.id.startsWith(iri));
```

Comment:
should we check if `foundKey` is `undefined` ?

Reply:
This would silently fails (correctly imo), but have updated it to be more explicit